src/components/Modals/Settings/constants.ts

```diff
@@ -0,0 +1,34 @@
+export const locales = [
```

Comment:
when we do the language switcher - move this closer to the core translations stuff

Reply:
put in a new utils.ts here https://github.com/shapeshift/web/tree/develop/src/assets/translations